docs/README.md

```diff
@@ -41,5 +41,10 @@ We are currently not accepting public contributions. However, we are accepting i
 * [Creating entities](content/create-entity.md)
 * [Entity checkout process](content/entity-checkout-process.md)
 * [The code generator](content/code-generator.md)
+* Testing
+  * [Testing overview](content/testing/testing-overview.md)
+  * [Testing guidelines](content/testing/testing-guidelines.md)
+  * [Running tests](content/testing/running-tests.md)
```

Comment:
Is this how to run tests or tests about something running? (Is 'running' an adjective or a verb?)